resin/resin/camera/first_person_camera_operator.cpp

```diff
@@ -0,0 +1,101 @@
+#include <glm/fwd.hpp>
+#include <glm/gtc/quaternion.hpp>
+#include <libresin/core/camera.hpp>
+#include <resin/camera/first_person_camera_operator.hpp>
+
+#include "resin/core/key_codes.hpp"
```

Comment:
```suggestion
#include <resin/core/key_codes.hpp>
```
remember to reformat